### DISCUSSION TITLE
Zephyr-os thinking about moving away from Breathe

### DISCUSSION THREAD

**Discussion by michaeljones** [michaeljones/breathe#754] [General]
Someone has kindly shared this with me. Sad to see a project moving away from Breathe but interesting to see why:

https://github.com/zephyrproject-rtos/zephyr/pull/39702

**Comment by michaeljones** [comment]
I've spoken to the author of the PR and they've said:

> the proposal was made for a few reasons, really:
> - Doxygen renders everything out of the box (no need to keep track of manually adding groups in the Sphinx side)
> - Sphinx in general is pretty slow when it comes to render a lot of content, (we see this issue with Kconfig pages)
> - As you mentioned incremental builds are also an issue, as of today breathe doesn't support them well

**Reply by michaeljones** [reply]
Thank you for sharing your thoughts @gmarull

**Reply by gmarull** [reply]
Note: a decision has not been made, it is just a proposal to analyze options.

**Comment by marc-hb** [comment]
Removed in August 2024
- https://github.com/zephyrproject-rtos/zephyr/pull/73671
